src/lib/formats/fs_vtech.cpp

```diff
@@ -310,6 +310,7 @@ std::pair<err_t, std::vector<u8>> vtech_impl::file_read(const std::vector<std::s
 	while(pos < len) {
 		if(track >= 40 || sector >= 16)
 			break;
+		fprintf(stderr, "Reading to pos %d from track %d sector %d\n", pos, track, sector);
 		auto dblk = m_blockdev.get(track*16 + sector);
```

Comment:
Is this extra `printf` intentional?  It looks like the kind of thing that would get spammy fast.

Reply:
It's a leftover from an unrelated debugging effort. I'll remove it.